src/duckling/game/drawable/shape-drawable.ts

```diff
@@ -0,0 +1,38 @@
+import {Drawable, DrawableType} from './drawable';
+import {Shape} from './shape';
+
+export interface ShapeDrawable extends Drawable {
+    shape: Shape;
+}
+
+export var defaultShapeDrawable : ShapeDrawable = {
```

Comment:
You can import the parent defaults and use immutableAssign to create a new object for the child.